_dprhtml/js/chrome.js

```diff
@@ -462,6 +462,10 @@ var DPR_Chrome = (function () {
     </div>`;
 
     $('#main-pane-container').append(`${html}`);
+      // Add event listeners for swipe gestures
+      document.getElementById('main-pane-container-section-' + sPos).addEventListener('touchstart', DPR_Gesture.touchStart, true);

```

Comment:
There is an API above to get the section ID - this is so we don't hardcode the prefix.

Also code appears out of alignment.

Reply:
Changed to use API.